index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    
+    <div class="container">
+    
+    <header class="portfolio-header">
+    <div class="logo">My Logo</div>
+    
+        <div class="menu-mobile">
+        <div class="menu-bar"></div>
+        <div class="menu-bar"></div>
+        <div class="menu-bar"></div>
+    </div>
+
+    </header> 
+  
+    
+
+<h1 class="salutation">
+   Hello. I’m Austin.
+    Glad to see you!</h1>
+<p class="developer">
+    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+</p>
+<h2 class="connect">LET'S CONNECT</h2>
```

Comment:
As mentioned by the previous reviewer, please update the styling of this text so that it is smaller and less bold and matches the figma design better🙏